src/libs/actions/Task.ts

```diff
@@ -113,6 +113,7 @@ function createTaskAndNavigate(
     assigneeAccountID = 0,
     assigneeChatReport?: OnyxEntry<OnyxTypes.Report>,
     policyID: string = CONST.POLICY.OWNER_EMAIL_FAKE,
+    shouldNavigate = true,
```

Comment:
IMO, I think we should rename the new param to 'isCreatedUsingMarkdown'

Reply:
I think `shouldNavigate` is more appropriate as it indicates what the function should do, as a param. `shouldNavigate` is more general as creating the task from markdown and avoid dismissing modal is just a specific case we are facing in this issue.